hostscripts/clouddata/syncSLErepos

```diff
@@ -99,7 +99,7 @@ rsync_and_unpack_iso()
 for version in 9 8 7 6; do
     # fetch latest code from version in development (not just latest beta)
     sync_from_ibs=
-    [ "$version" == "9" ] && sync_from_ibs=1
+    [ "$version" == "10" ] && sync_from_ibs=1
```

Comment:
Why change cloud 9//10 also? The commit message suggests that this is about Cloud 8.

Reply:
The commit message explains that.